pkgs/test_api/lib/backend.dart

```diff
@@ -2,6 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+@Deprecated('package:test_api is not intended for general use. '
```

Comment:
Do we see this one coming up in practice?

Reply:
Yes. I can't find it but there was a flutter bug recently where folks were importing from test_api and it caused unexpected issues.